.travis.yml

```diff
@@ -7,6 +7,7 @@ go: 1.6
 install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
+  - go get -u golang.org/x/net/context
   - export GO15VENDOREXPERIMENT=1
```

Comment:
We are using crypto which has dependency in this and build was failing due to this.